lisp/mastodon-tl.el

```diff
@@ -163,7 +237,11 @@ Return value from boosted content if available."
              (mastodon-tl--byline-author toot)
              (mastodon-tl--byline-boosted toot)
              " "
-             (format-time-string mastodon-toot-timestamp-format (date-to-time timestamp))
+             (let ((parsed-time (date-to-time timestamp)))
```

Comment:
If we don't use `timestamp` outside of defining `parsed-time` could we combine this `let` with the first `let` in this function? i.e., replace 

```elisp
(timestamp (mastodon-tl--field 'created_at toot))
```

with

 ```elisp 
(parsed-time (date-to-time (mastodon-tl--field 'created_at toot)))
```